src/CourseDashboard.cpp

```diff
@@ -82,3 +83,67 @@ void CourseDashboard::updateUser(User & user)
         break;
     }
 }
+
+std::string CourseDashboard::eMail()
```

Comment:
it should rather be called `provideEmail`

Reply:
General rule: method should have some verb (czasownik) in the name shortly describing whats method do